contrib/win32/win32compat/pwd.c

```diff
@@ -262,6 +262,50 @@ w32_getpwuid(uid_t uid)
 	return ret;
 }
 
+PSID
+getusid(void)
+{
+	HANDLE token = 0;
+	DWORD info_len = 0, length = 0;
+	TOKEN_USER* info = NULL;
+	PSID user_sid = NULL;
+
+	errno = 0;
+	if (OpenProcessToken(GetCurrentProcess(), TOKEN_QUERY, &token) == FALSE) {
+		debug2("OpenProcessToken() failed. Error code is : %d.\n", GetLastError());
+		errno = ENOENT;
+		goto done;
+	}
+	if (GetTokenInformation(token, TokenUser, NULL, 0, &info_len) == TRUE) {
+		debug2("GetTokenInformation() failed. Error code is : %d.\n", GetLastError());
+		errno = ENOENT;
+		goto done;
+	}
+	if ((info = (TOKEN_USER*)malloc(info_len)) == NULL){
+		debug2("GetTokenInformation() failed. Error code is : %d.\n", GetLastError());
```

Comment:
wrong comment. Also please make all of these debug3 (that the max level we are using in all of POSIX adapter code)